libsolidity/formal/CHC.cpp

```diff
@@ -873,7 +875,7 @@ void CHC::defineInterfacesAndSummaries(SourceUnit const& _source)
 						auto nondetPre = smt::nondetInterface(iface, *contract, m_context, 0, 1);
 						auto nondetPost = smt::nondetInterface(iface, *contract, m_context, 0, 2);
 
-						vector<smtutil::Expression> args{errorFlag().currentValue(), state().thisAddress(), state().state(1)};
+						vector<smtutil::Expression> args{errorFlag().currentValue(), state().thisAddress(), state().tx(), state().state(1)};
```

Comment:
Again, the previous blockchain state seems to be missing here unless I'm getting it all wrong...

Reply:
The previous is the one on this line, the `post state` is 3 lines below: `state().state(2)`

Reply:
Oh, right. It's there. It's fine then.